ci_framework/roles/edpm_prepare/tasks/main.yml

```diff
@@ -101,6 +101,24 @@
           --namespace={{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }}
           --for=jsonpath='{.status.phase}'=Complete --timeout=20m
 
+- name: Update OpenStack Services containers Env
+  when: cifmw_edpm_prepare_update_os_containers | bool
+  block:
+    - name: Generate OpenStack containers service env file
+      ansible.builtin.include_role:
```

Comment:
ditto
though, since the same role is called multiple times.... it may make sense to get include_role actually...